app-misc/vcontrold/files/vcontrold

```diff
@@ -0,0 +1,18 @@
+#!/sbin/openrc-run
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+PIDFILE="/run/vcontrold.pid"
+EXEC="/usr/sbin/vcontrold"
+
+start() {
+	ebegin "Starting vcontrold"
+	start-stop-daemon --start --quiet --pidfile $PIDFILE --exec $EXEC
```

Comment:
Any reason you redeclare `start()` and `stop()`? Please look into using the declarative form, i.e. just telling openrc what to execute.

Reply:
To be honest, I simply tried to write a working init script back then and it works. I tried to remove the start and stop function and to just add the command variable. It actually also starts vcontrold, but it won't stop it …

Reply:
Changed with a5b322ab782ecd52e82b41d3d648cc4eb22b0aca